### PR TITLE
Build tools once in CI

### PR DESCRIPTION
**Pull request by pjdufour-truss**
## Description

This PR uses the `workspace` in CircleCI (https://circleci.com/docs/2.0/concepts/) to only build a few tools once.  Given the new way Go's dependency management works, downstream jobs are wasting a lot of time downloading dependencies even if the tool doesn't even import them!  The `health-checker` and `renderer` tools are now called directly instead of rebuilt using `go run`.

## Reviewer Notes

Is there anything you would like reviewers to give additional scrutiny?
- Are there an adverse affects on developer workflow or local tooling?

## Setup

```
make clean
make build_tools
```

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/tree/master/docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/tree/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/tree/master/docs/database.md#zero-downtime-migrations))
  * [ ] Have been communicated to #dp3-engineering
  * [ ] Secure migrations have been tested using `scripts/run-prod-migrations`
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* Any new client dependencies (Google Analytics, hosted libraries, CDNs, etc) have been:
  * [ ] Communicated to @ntwyman
  * [ ] Added to the list of [network dependencies](https://github.com/transcom/mymove#client-network-dependencies)
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

None

## Screenshots

None

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -208,6 +208,11 @@ bin/ecs-deploy-task-container: server_deps server_generate
 bin/ecs-service-logs:
 	go build -i -ldflags "$(LDFLAGS)" -o bin/ecs-service-logs ./cmd/ecs-service-logs
 
+bin/renderer:
```
Comment (chrisgilmerproj) [inline Makefile]: I tried to keep these in alphabetical order by name, could you move it down?

Comment (pjdufour-truss) [reply Makefile]: done

**Comment by pjdufour-truss** [comment]
Adding `bin/chamber` and `bin/milmove` would be a whole new level of effort to "add" them to the workspace, too.  I'd rather make the fixes now.  Could create story for that.

**Comment by chrisgilmerproj** [comment]
> Adding `bin/chamber` and `bin/milmove` would be a whole new level of effort to "add" them to the workspace, too. I'd rather make the fixes now. Could create story for that.

Sure, let's make it a different story :)

File: Makefile [hunk 2]
```diff
@@ -339,11 +344,12 @@ build_tools: server_deps \
 	bin/compare-secure-migrations \
 	bin/ecs-deploy-task-container \
 	bin/ecs-service-logs \
+	bin/renderer \
```
Comment (chrisgilmerproj) [inline Makefile]: Same sorting here please:) But I'll still unblock.

**Pull request merged**